### PR TITLE
Use the built-in ESLint reporter (Stylish)

### PR DESCRIPTION
**Pull request by kasperisager**
This PR refactors Standard to report errors using the built-in reporter in ESLint.

Before:

screen shot 2015-08-03 at 19 46 37

After:

screen shot 2015-08-03 at 19 49 23

### CODE REVIEW

**Comment by jprichardson** [comment]
Not sure what @feross thinks about this PR, but presumably that's why he built [snazzy](https://github.com/feross/snazzy).

**Comment by Flet** [comment]
Unfortunately, I don't think this will get merged...
>"... the output format of standard is locked – it will never change" 
https://github.com/feross/standard/issues/142#issuecomment-111284255

And:
>"I would prefer to solve this outside of standard core, since it's not an essential feature.
I published a new package snazzy that does just what you need: it transforms "compact" output to "stylish"."
https://github.com/feross/standard/pull/202

I definitely recommend checking out [snazzy](https://github.com/feross/snazzy)!

**Comment by kasperisager** [comment]
Snazzy it is then!

**Comment by feross** [comment]
I'm afraid this change will break the editor tools that rely on the current output format on stdout.

We could try to detect if standard is running in a TTY (with `process.stdin.isTTY`) and use the stylish formatting in that case, falling back to the current formatting in scripts and when piping data, but this method is not foolproof.

**Comment by LinusU** [comment]
I would love to see some nicer output from standard, but I don't think that the extra effort (although small) of using snazzy outweighs that need.

I'm +1 on this getting merged thought, maybe with `isTTY`. But somehow I think that we have kind of the worst of two worlds currently. It's not very human readable and it's not very machine readable. I don't know, just my two cents...

**Comment by jprichardson** [comment]
It's best to just keep it the same. You can never predict how consumers of your software will use it. `isTTY` feels like a "non-pure" hack. Mocha uses this and it forced me to do some nasty things in `[electron-mocha](https://github.com/jprichardson/electron-mocha)`... but that's because they probably didn't anticipate a Node environment that's DOM-enabled.

**Comment by feross** [comment]
Thanks for the PR, but we don't need this feature in core.

There are many of modules that style the output - it's not that hard to parse. The built-in reporter works just fine, but there's also [snazzy](https://www.npmjs.com/package/snazzy), [standard-json](https://www.npmjs.com/package/standard-json), and [standard-tap](https://www.npmjs.com/package/standard-tap) for specific styles. Also [standard-reporter](https://www.npmjs.com/package/standard-reporter) supports a bunch more.
